Update Engine - Node

donavanbecker opened the pull request:
This PR is to update the node engine to be in line with ```hap-nodejs``` and their node engine.

donavanbecker commented:
> This is good, but wondering if we should set the minimum to be the current nodeJS LTS version?
> 
would we want to do ```latest``` or ```stable```

donavanbecker commented:
@NorthernMan54 I have updated it to be ```Node.js Version | v12.16.1```

donavanbecker commented on package.json:
```diff
@@ -21,7 +21,7 @@
   },
   "engines": {
-    "node": ">=10.17.0"
+    "node": ">=12.16.1"
```
@NorthernMan54 I have updated it to be ```Node.js Version | v12.16.1```

oznu commented:
Set the minimum in engines to v8.15.1. Everything still works on this version.

Update the readme to say v10+ as that's a "Current" LTS still.

oznu commented:
Forcing people to update Node.js more often than they have to will cause a spike in issues.

donavanbecker commented:
> Set the minimum in engines to v8.15.1. Everything still works on this version.
> 
> Update the readme to say v10+ as that's a "Current" LTS still.

@oznu I have updated the readme to say v10 and also changed the engine to v18.15.1.
